src/hardware/vga_dac.cpp

```diff
@@ -60,32 +60,80 @@ Note:  Each read or write of this register will cycle through first the
 
 enum { DacRead, DacWrite };
 
-static void vga_dac_send_color(const uint8_t palette_idx, const uint8_t color_idx)
+static bool is_cga_color(Rgb666 color)
```

Comment:
Might as well const these.

```
static bool is_cga_color(const Rgb666& color)
```

Reply:
Since Rgb666 contains only 3 bytes, a copy would be cheaper than a reference. Hence:
```suggestion
static bool is_cga_color(const Rgb666 color)
```

Reply:
Yeah, forgot the const.

Up to 32-bits it's cheaper to pass a copy, you're right. I think we even have a PVS Studio warning in place that will complain about primitives and small structs being passed as refs.

Reply:
Zero difference in a quick benchmark with 1 billion calls in a loop, but, fair enough, I guess.